### ISSUE TITLE
flan-t5-xxl text encoder too slow after `torch_neuronx.trace`

### ISSUE THREAD

**Issue by JH-ninjatech** [aws-neuron/aws-neuron-sdk#928]
Hi team,

I have traced a finetuned flan-t5-xxl `text_encoder` for my own pipeline. Before tracing, the calculation time for the below code is within 4s:
`text_embeddings = pipe.text_encoder(input_ids=input_ids, attention_mask=attention_mask)`
After I traced the model, the computation time for this increased to 15s.

**Comment by JH-ninjatech** [comment]
problem solved, turns out that I forgot to warmup the neuron instance.
